### ISSUE TITLE
For Kinetics-400 can't achieve the score in paper. ''Top 1 76.7''  ''Top 5 92.9''

### ISSUE THREAD

**Issue by FightAllDays** [facebookresearch/TimeSformer#16]
Hi, first of all, thank you for your amazing work for Video Understanding. I attempt to follow your work, but I can't achieve the best score in Kinetics-400 with the Code in this program. Is there anything that needs to be changed in the current version of the code for the best performance? Thank you again for your work!!!!

Best regards,
FightAllDays

**Comment by gberta** [comment]
It's difficult to tell where the issue might be without any further information. Other users were able to reproduce the results successfully. One potential reason might be that you are operating with an incomplete Kinetics-400 dataset. Did you make sure that most of the videos are present? Also, can you provide your training logs? It would be easier for me to debug if I have more information.

**Comment by FightAllDays** [comment]
Thank you very much for your reply!!!Attached is my training log. Could you please help me check it!!!
[TimeSformer(0-25).txt](https://github.com/facebookresearch/TimeSformer/files/6431366/TimeSformer.0-25.txt)
[TimeSformer(26-35).txt](https://github.com/facebookresearch/TimeSformer/files/6431367/TimeSformer.26-35.txt)

**Comment by gberta** [comment]
I took a quick glance at your logs. It seems that you validation set is much smaller than it's supposed to be (i.e., 13K instead of ~20K). That might be one of the reasons why you are not getting the same performance. Could you download the missing videos from https://github.com/cvdfoundation/kinetics-dataset? 

Also, did you run any inference experiments with one of our provided pre-trained models? If you cannot reproduce those results as well, that might indicate that this is a validation data issue indeed.

**Comment by FightAllDays** [comment]
OK, I will check the problem you mentioned as soon as possible, thanks for your kindly help!!!!!

**Comment by FightAllDays** [comment]
After checking the validation set again, it was found that the score could not be aligned because of missing data in part of the validation set. Thank you very much for your help, sorry for taking up your time!
